src/picomsg/src/oper_data_struct.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "../../libcharon/sa/ike_sa_state.h"
+#include <stdint.h>
+
+struct oper_data {
```

Comment:
The name "oper_data" is way to general. Rename to `terminator_tunnel_oper_data` or something similar.